editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -7370,6 +7370,54 @@ void Node3DEditor::_preview_settings_changed() {
 	}
 }
 
+void Node3DEditor::_picker_settings() {
+	// get default color picker mode from editor settings
```

Comment:
```suggestion
	// Get default color picker mode from editor settings.
```
Every comment should follow this style.

Although the comments in this PR aren't really necessary, as the code is rather self-explanatory.